docs/userguide/examples/library.py

```diff
@@ -0,0 +1,5 @@
+from parsl import App
+
+@App('python')
```

Comment:
So, `@App('python')` , `@App('python', dfk)` and `@App('python', dfk=DFK)` are all legal now ? This does look better.

Reply:
👍

Reply:
one more variable assignment to get @python

Reply:
Agreed, but let's wait on that for a separate PR